backend/server/server_utils.py

```
@@ -296,6 +375,9 @@ async def safe_run():
                 elif data.strip().startswith("human_feedback"):
                     logger.info(f"Processing human_feedback command")
                     running_task = run_long_running_task(handle_human_feedback(data))
+                elif data.strip().startswith("chat"):
+                    logger.info(f"Processing chat command")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)